epaint/src/bezier.rs

```diff
@@ -0,0 +1,1086 @@
+use std::ops::Range;
```

Comment:
To silence the clippy warnings:

```suggestion
#![allow(clippy::many_single_char_names)]

use std::ops::Range;
```

Reply:
Thank you so much. I  fixed all Clippy warnings and learned a lot. :-)